frontend/actions.js

```diff
@@ -1,11 +1,11 @@
-import Cardano from '../wallet/cardano-wallet'
-import {CARDANOLITE_CONFIG} from './config'
-import {
+const generateMnemonicWords = require('../wallet/mnemonic').generateMnemonic
```

Comment:
"mnemonicWords" is a bit redundant since the mnemonic itself is defined as a set of words. I would name it simply "generateMnemonic"